### ISSUE TITLE
Freezes for german since v6.0.0

### ISSUE THREAD

**Issue by ahippler** [valentjn/vscode-ltex#68]
**Describe the bug**
Freezes after a couple of sentences with language "de-DE" since the update to v6.0.0.
Opening lager (more the 20 lines) files shows no errors (but there are some). 
Restarting the language server does not solve the problem.

**Steps to reproduce**
- language "de-DE"
- write text or open lager files
- ltex stops working

**Sample document**

Working:
```
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula
eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient
montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque
eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo,
fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut,
imperdiet a, venenatis vitae, justo.
```

Stopps working:
```
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula
eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient
montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque
eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo,
fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut,
imperdiet a, venenatis vitae, justo. Nullam
```

**LTeX configuration**

<details>

```
  "ltex.language": "de-DE",
```

</details>

**"LTeX Language Server" log file**

<details>

```
[Info  - 3:35:37 PM] Starte ltex-ls ...
Juli 02, 2020 3:35:40 PM org.bsplines.ltexls.LtexLanguageServer initialize
INFORMATION: Setting locale to de
Juli 02, 2020 3:35:46 PM org.bsplines.ltexls.DocumentChecker checkAnnotatedTextFragment
INFORMATION: Checking the following text in language de-DE via LanguageTool: "..."... (truncated to 100 characters)
Juli 02, 2020 3:35:49 PM org.bsplines.ltexls.DocumentChecker checkAnnotatedTextFragment
INFORMATION: Obtained 11 rule matches
Juli 02, 2020 3:36:04 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNUNG: Unmatched cancel notification for request id 2
```

</details>

**"LTeX Language Client" log file**
shows no difference between 5.0.2 and 6.0.0

**Version information**
List here the version information of the relevant software.

- Operating system: macOS 10.15.5
- VS Code: 1.46.1
- vscode-ltex: 6.0.0
- Java: openjdk version "14.0.1" 2020-04-14

**Additional context/information**
Using diffrent java version does not help. 
Changing the language to english and killing the java process solves the problem.
As a workaround i downgraded to vscode-ltex 5.0.2 which is working fine with the same documents.

**Comment by valentjn** [comment]
Please include a sample document in your bug report as instructed.

**Comment by ahippler** [comment]
Maybe it stops working when it detects too many errors? It looks like the number of errors makes the difference not the length of the text.

**Comment by valentjn** [comment]
I think this is caused by https://github.com/languagetool-org/languagetool/issues/3181. LanguageTool 5.0 writes random stuff to stdout and that messes up the LSP communication, resulting in a deadlock.

**Comment by valentjn** [comment]
Implemented a workaround in 6.0.1. Thanks for the report.

**Comment by ahippler** [comment]
Thanks for the quick fix
